cli/create.go

```diff
@@ -29,6 +29,17 @@ const PresetNone = "none"
 
 var ErrNoPresetFound = xerrors.New("no preset found")
 
+// isNonInteractive checks if the command is running in non-interactive mode
+// (i.e., the --yes/-y flag was provided).
+func isNonInteractive(inv *serpent.Invocation) bool {
```

Comment:
This should probably go into the cliui package beside the SkipPromptOption function